cmd/remove-members/main.go

```diff
@@ -0,0 +1,228 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	flag "github.com/spf13/pflag"
+	"k8s.io/test-infra/prow/config/org"
+	"sigs.k8s.io/yaml"
+)
+
+type options struct {
+	confirm    bool
+	configPath string
+}
+
+func parseOptions() options {
+	var o options
+	flag.StringVar(&o.configPath, "path", ".", "Path to config directory/subdirectory")
+	flag.BoolVar(&o.confirm, "confirm", false, "Modify the actual files or just simulate changes")
+
+	flag.Parse()
+
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "Usage: remove-members [--confirm] [--path] member-file (file-containing-members-list)\n")
+		flag.PrintDefaults()
+	}
+
+	if len(flag.Args()) != 1 {
+		flag.Usage()
+		os.Exit(1)
+	}
+
+	return o
+}
+
+func main() {
+
+	o := parseOptions()
+
+	memberList, err := readMemberList(flag.Args()[0])
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err = removeMembers(memberList, o); err != nil {
+		log.Fatal(err)
+	}
+
+}
+
+//readMemberList reads the list of members to be removed from the given filepath
+func readMemberList(path string) ([]string, error) {
+	file, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+
+	var members = strings.Split(string(file), "\n")
+	return members, nil
+}
+
+//removeMembers walks through the config directory and removes the occurences of the given member name
+func removeMembers(memberList []string, o options) error {
+	for _, member := range memberList {
+		var orgs, teams []string
```

Comment:
This will redeclare orgs and teams with each iteration, better to move it outside for loop.